pyproject.toml

```diff
@@ -13,33 +13,42 @@ classifiers = [
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9"
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12"
 ]
 
 
 [tool.poetry.scripts]
 osia = 'osia.cli:main_cli'
 
 [tool.poetry.dependencies]
-python = "^3.8"
-openstacksdk = "*"
+beautifulsoup4 = "*"
 boto3 = "*"
-jinja2 = "*"
+coloredlogs = "*"
 dynaconf = {extras = ["yaml"], version = "*"}
 gitpython = "*"
-beautifulsoup4 = "*"
-coloredlogs = "*"
-urllib3 = "<2" # add temporarily to enable dependency resolution in real time
+jinja2 = "*"
+openstacksdk = "*"
+python = "^3.8"
 semantic-version = "^2.10.0"
+urllib3 = "<2" # add temporarily to enable dependency resolution in real time
```

Comment:
Is this still needed?

Reply:
Yeah I can take a look and include depenbot bumps as well

Reply:
Well the dependabot bumps are related only to the lock file - since you bumped the python version, I see you also relocked, so everyting should be to the latest possible version available. I'd just try to change the urllib to "*" and see if it locks in finite time :)